tests/rpc/gossamer_utils.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package rpc
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/rpc/modules"
+)
+
+var (
+	keyList = []string{"alice", "bob", "charlie", "dave", "eve", "fred", "george", "heather"}
+)
+
+// RunGossamer will start a gossamer instance and check if its online and returns CMD, otherwise return err
+func RunGossamer(t *testing.T, nodeNumb int, dataDir string) (*exec.Cmd, error) {
+
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+
+	gossamerCMD := filepath.Join(currentDir, "../..", "bin/gossamer")
+
+	//var cmd *exec.Cmd
+	//cmd = exec.Command(gossamerCMD, "init",
```

Comment:
will this part be re-added once the issue is addressed?

Reply:
yes I fixed it now, please re-check